internal/yggdrasil/yggdrasil.go

```diff
@@ -446,9 +447,16 @@ func (h *Handler) toWorkloadList(ctx context.Context, logger logr.Logger, deploy
 			data = &models.DataConfiguration{Paths: paths}
 		}
 
+		podmanLabels := labels.GetPodmanLabels(deployment.Labels)
+		if !labels.ValidatePodmanLabels(podmanLabels, deployment) {
```

Comment:
2 thoughts about this one:
1. We can move the validation from here to the validating webhook of edge deployment. In that case, we won't allow user to create/update incorrect edgedeployment.
2. We don't need to record anything (event/log) in this context or perform any checks here. Else, if the deployment isn't valid, we'd might better return an error or filter the problematic deployment.
But the problem of 2 can be saved if we go with 1.
WDYT?